apm-server.docker.yml

```diff
@@ -214,7 +214,8 @@ apm-server:
       #password: "changeme"
 
   #kibana:
-    # Enabled must be true to enable APM Agent configuration, and for fetching source maps uploaded through Kibana.
+    # Required if running APM Server standalone to check if the APM package has been installed. Also
+    # required for APM agent configuration when using an output other than Elasticsearch.
```

Comment:
Strictly speaking it is required when apm-server.agent.config.elasticsearch is not set AND output.elasticsearch is not valid (not present / not enough privileges). I believe that's what you were trying to express as well. Is there a concise way to convey that?

Reply:
Thanks, I'll clarify.